app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsActivity.java

```diff
@@ -50,20 +54,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_event_details);
 
-        ButterKnife.bind(this);
+        Event initialEvent = getIntent().getParcelableExtra(EVENT_KEY);
+
+        if(initialEvent == null) {
```

Comment:
It does not have id. The case will be when programmer forgets to pass the event to the activity. We can't trust anyone. Showing a blank screen is worse than crashing the app. At least second case tells you that something is wrong.

If I forget to add event as intent extra, it will give NPE, so I have to check if it is indeed passed and log and exit if it's not